### DISCUSSION TITLE
特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by relaxman86** [Yowkees/keyball#273] [Q&A]
赤●のキーが反応しません。ダイオードのどの部分のはんだ不良が原因になってきますでしょうか。

![PXL_20230711_130059815 MP~2](https://github.com/Yowkees/keyball/assets/138578615/2d499fbb-b3f8-4beb-8ea0-c04f4389ee51)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

該当のダイオードは添付写真の緑丸で示したものになります。
ただ、ダイオードよりキーソケット自体の半田付けミスが多いので、こちらも再度ご確認ください。

以上、お手数ですがご対応お願いいたします。

![KB61R_keyfix](https://github.com/Yowkees/keyball/assets/84209772/a0ec9e61-749a-4aef-bdaa-654f25feb09e)
